SmartPlaces.Facilities/samples/Topology/README.md

```diff
@@ -49,3 +49,102 @@ The following secrets must be defined in the KeyVault in order to run this code:
 ## Logging and Metrics
 
 Logs and Metrics will appear in the Application Insights instance pointed to by the App Insights connection string. For more information on how to configure the Application Insights SDK, please see: https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service
+
+There are three common ways to monitor / debug the topology loading process:
+
+1. If you deployed the [dashboards](../deploy/deploy.ps1), you should be able to review various graphs to help identify issues. There are 4 types of graphs currently available:
```

Comment:
Github is apparently case-sensitive on this path. If I click as is it 404s, however, if I edit the url of where I'm sent to `/Deploy/deploy.ps1` I am able to access the file.